terraform/stacks/main/stack.tf

```
@@ -305,6 +305,15 @@ module "diego" {
   log_bucket_name = module.log_bucket.elb_bucket_name
 }
 
+module "opensearch" {
```

```suggestion
module "opensearch_logs_customer" {
```

We also need to make this module conditional on a boolean because we only want to test it in dev for now and not deploy in stage/prod. Here's an example of how to make a resource conditional:

https://github.com/cloud-gov/cg-provision/blob/df069274dbd9b6b0e5882c9ecb0c56143c82ac41/terraform/modules/iam_role/role.tf#L13